### PR TITLE
HIVE-24408: Upgrade Parquet to 1.11.1

### PR DESCRIPTION
**Pull request by sunchao**
### What changes were proposed in this pull request?

Upgrade Parquet version from 1.11.0 to 1.11.1

### Why are the changes needed?

Parquet 1.11.1 has some important bug fixes so it would be beneficial to upgrade to it.

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

N/A

### CODE REVIEW

**Comment by dongjoon-hyun** [comment]
Thank you so much, @sunchao !

File: ql/src/test/org/apache/hadoop/hive/ql/io/parquet/TestHiveSchemaConverter.java [hunk 1]
```diff
@@ -431,7 +431,8 @@ public void testMapOriginalType() throws Exception {
     Type secondLevel = topLevel.asGroupType().getFields().get(0);
     // there is one repeated field for mapCol, the field name is "map" and its original Type is
     // MAP_KEY_VALUE;
-    checkField(secondLevel, "map", Repetition.REPEATED, LogicalTypeAnnotation.MapKeyValueTypeAnnotation.getInstance());
```
Comment (dongjoon-hyun) [inline ql/src/test/org/apache/hadoop/hive/ql/io/parquet/TestHiveSchemaConverter.java:434]: BTW, do you know which Parquet JIRA made this change?

Comment (sunchao) [reply ql/src/test/org/apache/hadoop/hive/ql/io/parquet/TestHiveSchemaConverter.java:434]: I don't know yet. Checked all the JIRAs in 1.11.1 and it's not obvious which one caused it. I'll post my finding later.

Comment (dongjoon-hyun) [reply ql/src/test/org/apache/hadoop/hive/ql/io/parquet/TestHiveSchemaConverter.java:434]: Thanks!

**Review by dongjoon-hyun — APPROVED** [review]

**Comment by dongjoon-hyun** [comment]
Do you think we can merge this to `master`, @sunchao ?

**Comment by sunchao** [comment]
@dongjoon-hyun yes we'll need this to be approved by a committer first though - @jcamachor could you review this? since you have done something similar earlier by upgrading Parquet to 1.10.0

**Review by jcamachor — APPROVED** [review]

**Comment by dongjoon-hyun** [comment]
Thank you, @jcamachor and @sunchao !

**Comment by sunchao** [comment]
Thanks @jcamachor for merging!

**Pull request merged**